db/migrate/20170209184350_add_reply_to_statuses.rb

```diff
@@ -2,7 +2,7 @@
 
 class AddReplyToStatuses < ActiveRecord::Migration[5.0]
   def up
-    add_column :statuses, :reply, :boolean, nil: false, default: false
```

Comment:
I think this `nil` was a typo for `null: false` probably not noticed when this was added - but a future migration does correct this and change the column nulls value